src/app.js

```diff
@@ -32,6 +35,14 @@ if (dsn) {
   })
 }
 
+if (connectionString) {
+  const appInsights = new ApplicationInsights({
+    config: { connectionString },
+  })
+  appInsights.loadAppInsights()
+  appInsights.trackPageView()
```

Comment:
Not sure if we want to turn this on? We have `autoSessionTracking: false` for Sentry as well.